### PR TITLE
Replace legacy docker-compose

### PR DESCRIPTION
**Pull request by xsalonx**
#### I DON'T have JIRA ticket
- [x] explain what this PR does
- [ ] if it is a new feature, explain how you plan to use it
- [ ] tests are added
- [ ] documentation was updated or added

Notable changes for users:
- NA

Notable changes for developers:
- Replaces `docker-compose` command by `docker compose`
- Updates files name as per docker requirements: https://docs.docker.com/compose/project-name/#set-a-project-name 

Changes made to the database:
- NA

### CODE REVIEW

File: .github/workflows/bookkeeping.yml [hunk 1]
```diff
@@ -17,20 +17,20 @@ jobs:
       matrix:
         test_type: [unit,
                     api,
-                    lhcPeriods,
```
Comment (martinboulais) [inline .github/workflows/bookkeeping.yml:20]: Why this change? I think it follows our js file name convention, even if I prefer your way I think we should keep the original.

**Review by martinboulais — DISMISSED** [review]

File: .github/workflows/bookkeeping.yml (hunk 1 above)
Comment (xsalonx) [reply .github/workflows/bookkeeping.yml:20]: I thought it resolves issue with hanged jobs: 
image
, but apparently no

Comment (xsalonx) [reply .github/workflows/bookkeeping.yml:20]: but following instruction is `"COMPOSE_PROJECT_NAME=${{ matrix.test_type }}_app"` `test_type` cannot has upercase letters

Comment (graduta) [reply .github/workflows/bookkeeping.yml:20]: @xsalonx is referring to :
https://docs.docker.com/compose/project-name/#set-a-project-name

**Review by graduta — APPROVED** [review]

File: .github/workflows/bookkeeping.yml (hunk 1 above)
Comment (graduta) [reply .github/workflows/bookkeeping.yml:20]: The `hanging jobs` are due to the fact that GH PR rules are waiting for those jobs names but because you are changing them, they are never being created.
I will update the rules accordingly
